test/functional/test_framework/authproxy.py

```diff
@@ -76,6 +77,7 @@ def __init__(self, service_url, service_name=None, timeout=HTTP_TIMEOUT, connect
         passwd = None if self.__url.password is None else self.__url.password.encode('utf8')
         authpair = user + b':' + passwd
         self.__auth_header = b'Basic ' + base64.b64encode(authpair)
+        self.timeout = timeout
 
         if connection:
             # Callables re-use the connection of the original proxy
```

Comment:
I think this is no longer supported? So could remove the named arg "connection"?